index.html

```diff
@@ -0,0 +1,19 @@
+<html>
+  <head>
+    <title>Hello World</title>
+  
+    <script src="node_modules/core-js/client/shim.min.js"></script>
+    <script src="node_modules/zone.js/dist/zone.min.js"></script>
+  
+  </head>
+  
+  <body>
+    
+    <app-component></app-component>
+
+  <script src="bazel-out/darwin-fastbuild/bin/src/rollup.runfiles/angular_bazel_example/dist/bundle.js"></script>
```

Comment:
I suppose this example ought to come with a <10 line express server so that the bundle can be served from the same URL as it would be under `ts_devserver`

Reply:
My trick is just to have a ts_devserver which doesn't actually produce a bundle itself but instead lists the production bundle as a static resource.

Reply:
I have added configuration to match the url of `ts_devserver`